cluster/gce/gci/configure-helper.sh

```diff
@@ -682,15 +685,15 @@ function start-kubelet {
     flags+=" --enable-debugging-handlers=false"
     flags+=" --hairpin-mode=none"
     if [[ "${REGISTER_MASTER_KUBELET:-false}" == "true" ]]; then
-      flags+=" --api-servers=https://${KUBELET_APISERVER}"
+      flags+=" --kubeconfig=/var/lib/kubelet/kubeconfig"
```

Comment:
Default file path for this was a mistake and led to the need for flags like --require-kube-config

We should be explicit